bedrock/firefox/templates/firefox/welcome/page14.html

```diff
@@ -0,0 +1,98 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ ftl('welcome-page14-page-title-alt') }}{% endblock %}
+
+{% block page_image %}{{ static('img/products/vpn/common/social-share.png') }}{% endblock %}
+
+{% block body_class %}welcome-page14{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page10') }}
+  {{ css_bundle('firefox_welcome_page14') }}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle("firefox_welcome_page14") }}
```

Comment:
Issue (blocking): this script will throw an error when `SWITCH_VPN_MOMENTS_COUPON_CODE=off`, so suggest wrapping it in a conditional:

```suggestion
  {% if show_coupon %}
    {{ js_bundle("firefox_welcome_page14") }}
  {% endif %}
```